search/templates/search/yang_tree.html

```diff
@@ -124,8 +124,14 @@ <h3>{{ title }}</h3>
               },
               {
                 header: '<b>Path</b>',
+                maxWidth: 200,
```

Comment:
200 seemes a bit much. I would go with 50 maybe and people can resize it if they want check it out on the page.. it is implemented